.github/workflows/conformance-multicluster-v1.10.yaml

```diff
@@ -147,9 +147,12 @@ jobs:
             --relay-version=${SHA}"
           CLUSTERMESH_ENABLE_DEFAULTS="--apiserver-image=quay.io/${{ github.repository_owner }}/clustermesh-apiserver-ci \
             --apiserver-version=${SHA}"
+          CONNECTIVITY_TEST_DEFAULTS="--base-version=v1.10 \
+            --flow-validation=disabled"
```

Comment:
Why are we disabling flow validation here?

Reply:
Mindless copy&paste, it seems. Reverted that change.